### ISSUE TITLE
Reactify issue.

### ISSUE THREAD

**Issue by DylanPiercey** [bevacqua/react-dragula#8]
Not exactly sure why, maybe it's browserify bug, but for some reason when I try to browserify and this module is included it will attempt to run the "reactify" transform. Since "reactify" is a devdependency and I use babel instead it doesn't find it and errors out.

``` console
Cannot find module 'reactify' from '/.../node_modules/react-dragula'
```

A quick fix would just be making reactify a regular dependency or not using the package.json browserify fields.

**Comment by razh** [comment]
Ah sorry, this is my fault. You can pin to 1.1.0 for the time being.

**Comment by bevacqua** [comment]
Released #9 as `1.1.13`
